seed/src/main.rs

```diff
@@ -153,6 +155,9 @@ pub struct Options {
     /// max number of passive members to set in [`membership::Params`].
     #[argh(option, default = "membership::Params::default().max_passive")]
     pub membership_max_passive: usize,
+
+    #[argh(option, description = "path to the secret key file")]
```

Comment:
```suggestion
    /// path to the secret key file
    #[argh(option)]
```
Let's stay consistent with using the doc strings as the description